coverage-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -4,6 +4,6 @@
 #Fri Dec 22 14:41:56 CET 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
Why downgrade gradle?

Reply:
Unfortunatly for gradle 8.5 I got the following error during the github build and test jobs
```
Exception is:
java.lang.NoClassDefFoundError: org/gradle/api/publish/ivy/internal/publication/DefaultIvyPublicationIdentity
``` 
This is why I downgraded the versions. Maybe there is a problem with github actions and gradle 8.5 (which was released end of november 2023) because when I run build locally I got no errors.

Reply:
It didn't happen with the coverage reporter project though right? it was the intellij plugin whose jobs were failing

Reply:
Yes exactly. And the coverage-plugin after I added the build-and-test.yml

Reply:
Ok I will change it